packages/glimmer-runtime/lib/builder.ts

```diff
@@ -279,6 +279,14 @@ export class ElementStack {
     return fragmentBounds;
   }
 
+  insertTextBefore(nextSibling: Node, text: string): Fragment {
```

Comment:
which probably means we need a better name for than `insertTextBefore` at this level – the point is we are inserting some "dynamically bounded" content that happens to be text at the moment, but that might change later